packages/react/src/editor/BlockNoteDefaultUI.tsx

```diff
@@ -48,10 +48,10 @@ export function BlockNoteDefaultUI(props: BlockNoteDefaultUIProps) {
       {editor.tableHandles && props.tableHandles !== false && (
         <TableHandlesController />
       )}
-      {editor.comments && props.comments !== false && (
+      {editor.comments && (
         <>
           <FloatingComposerController />
-          <FloatingThreadController />
+          {props.comments !== false && <FloatingThreadController />}
```

Comment:
I'm not sure about this. With this change, users cannot use their own composer should they want to